frontend/public/components/_monitoring.scss

```diff
@@ -47,3 +47,10 @@ $monitoring-line-height: 18px;
 .btn--silence-add-more {
   padding: 0;
 }
+
+.monitoring-state-icon {
+  width: 15px;
```

Comment:
Are we able to use `fa-fw` for fixed width icons?

https://fontawesome.com/v4.7.0/examples/

Reply:
Yes, I think using `fa-fw` would be better. It does mean using `fa-fw` for a `pficon` icon in one case, but I think that is OK. Thanks!

Reply:
Yeah `fa-fw` should work ok for `pficon` icons

Reply:
👍 I made that change